modules/all-modules/pom.xml

```diff
@@ -15,6 +15,12 @@
     </description>
 
     <dependencies>
+        <!-- TODO(gfernandes) added due to HTTP extension needs to access the Sockets extension API, remove this once MULE-9867 is fixed  -->
```

Comment:
I don't understand why you said. Comment says "remove this once MULE-9867 is fixed". MULE-9867 is alredy fixed, then or you reference a different jira issue or remove the comment and the workaround